main.py

```diff
@@ -1,10 +1,52 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+        for player_name, player_data in players_data.items():
+            race_data = player_data["race"]
+            if race_data:
+                race_name = race_data["name"]
+                race_description = race_data["description"]
+                race, _ = Race.objects.get_or_create(
+                    name=race_name,
+                    description=race_description
+                )
+
+                skills_data = race_data["skills"]
+                for skill_data in skills_data:
+                    skill_name = skill_data["name"]
+                    skill_bonus = skill_data["bonus"]
+                    Skill.objects.get_or_create(
+                        name=skill_name,
+                        bonus=skill_bonus,
+                        race=race
+                    )
+            else:
```

Comment:
you can move race assignment above line 12 so you don't have to use extra `else` statement